src/schema/search.ts

```diff
@@ -312,6 +337,28 @@ export const resolvers: IResolvers<unknown, Context> = traceResolvers({
         query: args.query,
       };
     },
+    searchTagSuggestions: async (
```

Comment:
I noticed this the other day why do we use keyword and tag differently 👀

Reply:
it's a legacy reason. At first we had tags and then i wanted to change the way i manage and discover the tags without affecting everything so I had to come up with a new name, keyword. Everything else was already using the tags term so I kept it. 🤷 
Bad decisions

Reply:
Ah so moving forward can we just start using keyword everywhere 👍

Reply:
I'll refactor so it is called keyword here, on frontend though we still use Tag almost everywhere.

Reply:
I don't mind keeping it tags everywhere

Reply:
@rebelchris gonna merge this as is, let's sync on it tomorrow when you are back.